raster-foundry/raster-foundry
# Add batch environment for hitl processing

## Overview

Adds environment for running python code within RV docker container. Includes placeholder script that takes a "project id" and just prints it out

to test:

run to build 
```
docker-compose build batch-hitl
```

then run placeholder scrips
```
docker-compose run batch-hitl python hitl/run_hitl.py 1234567
```
output
```
Creating raster-foundry_batch-hitl_run ... done
processing project: 1234567
```


### Checklist


### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

- How to test this PR
- Prefer bulleted description
- Start after checking out this branch
- Include any setup required, such as bundling scripts, restarting services, etc.
- Include test case, and expected output

Closes #https://github.com/azavea/foss4g-2022-hitl-groundwork-rastervision/issues/15
